### PR TITLE
Ks checksum

### PR DESCRIPTION
**Pull request by nwatx**

### CODE REVIEW

**Comment by nwatx** [comment]
Just a note, the MST.problems was copy pasted from guide - should be the exact same.

**Comment by dongliu0426** [comment]
indentation is kinda messed up
Screen Shot 2021-05-24 at 11 51 26 AM

**Comment by nwatx** [comment]
> indentation is kinda messed up
> Screen Shot 2021-05-24 at 11 51 26 AM

should be fixed now :p

File: content/4_Gold/MST.problems.json [hunk 1]
```diff
@@ -36,23 +31,37 @@
...
+      "tags": ["MST"],
+      "solutionMetadata": { "kind": "autogen-label-from-site", "site": "CF" }
     },
     {
       "uniqueId": "cf-1513D",
       "name": "GCD and MST",
       "url": "https://codeforces.com/problemset/problem/1513/D",
       "source": "CF",
-      "difficulty": "Easy",
+      "difficulty": "Normal",
```
Comment (dongliu0426) [inline content/4_Gold/MST.problems.json:61]: wait why was this changed

Comment (nwatx) [reply content/4_Gold/MST.problems.json:61]: I copy pasted it. Apparently it is also normal on deployment.
image

**Review by dongliu0426 — APPROVED** [review]

**Review by thecodingwizard — CHANGES_REQUESTED** [review]
@nwatx it would be preferable to not use Ben's template in Gold (Plat is proooooobably fine). See https://github.com/cpinitiative/usaco-guide/issues/806 for the "acceptable" templates.

(But if you don't want to change it, it's fine :P)

**Comment by nwatx** [comment]
> @nwatx it would be preferable to not use Ben's template in Gold (Plat is proooooobably fine). See #806 for the "acceptable" templates.
> 
> (But if you don't want to change it, it's fine :P)
> 
> Also, merge conflict :(

merge should be fixed now i think

**Comment by dongliu0426** [comment]
there's still a merge conflict
